source/ProtonPack/ProtonPack.ino

```diff
@@ -226,6 +236,7 @@ void loop() {
         b_2021_ramp_up = false;
         b_2021_ramp_up_start = false;
         b_inner_ramp_up = false;
+        b_fade_out = true;
```

Comment:
This only appears to affect the 2021 mode, was that correct? I did notice it during the shutdown but wasn't sure if there was an intended impact to the 84/89 modes as I didn't notice anything different.

Reply:
Correct, 2021